autoware_launch/launch/components/tier4_perception_component.launch.xml

```diff
@@ -21,6 +21,7 @@
       name="object_recognition_detection_pointcloud_map_filter_param_path"
       value="$(find-pkg-share autoware_launch)/config/perception/object_recognition/detection/pointcloud_map_filter.param.yaml"
     />
+    <arg name="object_recognition_detection_fusion_sync_param_path" value="$(find-pkg-share autoware_launch)/config/perception/object_recognition/detection/roi_sync.param.yaml"/>
```

Comment:
Along with https://github.com/autowarefoundation/autoware.universe/pull/3713#discussion_r1199561768
```suggestion
    <arg name="fusion_roi_sync_param_path" value="$(find-pkg-share autoware_launch)/config/perception/object_recognition/detection/roi_sync.param.yaml"/>
```

Reply:
I modeled it on other variable names near it, such as [this](https://github.com/autowarefoundation/autoware_launch/blob/b27e2f3cae2442d37c032d61d33f2c2ac5d8a13f/autoware_launch/launch/components/tier4_perception_component.launch.xml#L17) and [that](https://github.com/autowarefoundation/autoware_launch/blob/b27e2f3cae2442d37c032d61d33f2c2ac5d8a13f/autoware_launch/launch/components/tier4_perception_component.launch.xml#L21). They are very long, too.